web/thesauruses/cplusplus/control_structures.json

```diff
@@ -0,0 +1,114 @@
+{
+  "meta": {
+    "language": "language_id",
```

Comment:
Can you add the language, version, and friendly name to the top of this file? It help with correctly loading the data on the site.